scripts/ecnconfig

```diff
@@ -21,9 +21,9 @@ optional arguments:
   -ymax  --yellow-max          set max threshold for packets marked yellow
   -rmin  --red-min             set min threshold for packets marked red
   -rmax  --red-max             set max threshold for packets marked red
-  -gdrop --green-drop-prob     set drop probability for packets marked green
-  -ydrop --yellow-drop-prob    set drop probability for packets marked yellow
-  -rdrop --red-drop-prob       set drop probability for packets marked red
+  -gdrop --green-drop-prob     set max probability for packets marked green
```

Comment:
confusing, if parameter is "drop probability", why changing comment to "max probability"?

Reply:
maybe "max drop probability"?

Reply:
@andriymoroz-mlnx  According to RED algorithm (e.g., Figure 5 in https://conferences.sigcomm.org/sigcomm/2015/pdf/papers/p523.pdf), this parameter determines the maximum probability when the queue size is between Kmin (gmin) and Kmax (gmax). RED can be used to mark packets or drop packets.

Reply:
@andriymoroz-mlnx  maybe "max drop/marking probability ..."

Reply:
right it can also mark...
ok, max drop/mark probability sounds good for me

Reply:
@andriymoroz-mlnx  I think gdrop is a little bit misleading. At the first glance, I think it indicates if dropping green packets (instead of ECN marking). g-max-prob is a much better name. But if we change gdrop in ecnconfig, we may need to change many other places. So let's keep it here.

Reply:
not sure it is used anywhere, but agree